os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -80,6 +80,11 @@ config AMEBASMART_SPI1
 config AMEBASMART_SPI_EXCHANGE
 	bool "Enable AMEBASMART SPI EXCHANGE"
 	default y
+config AMEBASMART_SPI_DMA
+	bool "SPI DMA"
+	default n
+	---help---
+		Use DMA to improve SPI transfer performance.  Cannot be used with CONFIG_AMEBASMART_SPI_INTERRUPT.
```

Comment:
Where is the `AMEBASMART_SPI_INTERRUPT`?

Reply:
It is not implemented yet, but i saw this macro in amebasmart_spi.c, should i remove it?

Reply:
I have removed it currently, if future interrupt is implemented, i will add back the comment.